docker_for_beginner/build_docker_image_from_a_running_container.md

```diff
@@ -0,0 +1,118 @@
+# Build Docker Image From A Running Container
```

Comment:
General Comments:

- Use --name to assign name to container and use that name in all the commands
- Mention the little bit of command information like wget will also download the file